README.md

```diff
@@ -12,6 +12,7 @@ Please read [contribution guidelines](contributing.md), thank you!
 - [Web Development](#web-development)
 - [Mobile Development](#mobile-development)
 - [Game Development](#game-development)
+- [AI / Machine Learning / Data Science](#ai---machine-Learning---data-science)
```

Comment:
It seems should be `#ai--machine-learning--data-science` here 👀

Reply:
Thanks, yes. I changed it.